src/App.tsx

```diff
@@ -1,25 +1,46 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [arrItems, setArrayItems] = useState<string[]>([]);
+  const [perPage, setPerPage] = useState<number>(5);
+  const [currentPage, setCurrentPage] = useState<number>(1);
+
+  useEffect(() => {
+    setArrayItems(items);
+  }, []);
+
+  useEffect(() => {
+    setCurrentPage(1);
+  }, [perPage]);
```

Comment:
The dependency array for this useEffect hook includes 'perPage', but it doesn't use any other state or props. If 'perPage' is intended to reset the current page when it changes, consider adding a comment to clarify this intention, as it may not be immediately obvious why this effect is needed.